### PR TITLE
Update how-to-create-release-announcements.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Minor formatting fixes.

### CODE REVIEW

File: operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md [hunk 1]
```diff
@@ -93,10 +93,10 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 ## Language Guidelines
 
 1. Use simple language with no buzzwords
-2. Specify Desktop app or Mobile App if a feature is not dependent on the server release
+2. Specify Desktop app or Mobile app if a feature is not dependent on the server release
```
Comment (jasonblais) [inline operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md:96]: I forget if we decided to use "Desktop App" and "Mobile App" when speaking about the Mattermost apps themselves?

Comment (justinegeffen) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md:96]: I've been using "Desktop app" and "Mobile app." My vote is for "Mattermost Desktop app" and "Mattermost Mobile app" and I have updated this PR: https://github.com/mattermost/mattermost-handbook/pull/362 so we have a starting point.

Comment (jasonblais) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md:96]: @justinegeffen Added a comment in #362: https://github.com/mattermost/mattermost-handbook/pull/362#pullrequestreview-492809359

Comment (justinegeffen) [reply operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md:96]: Leaving this as-is for now as it's consistent within the doc. Based on all the conversations around this we're leaning toward Mobile App, Desktop App, and Web App for now.

**Review by cwarnermm — APPROVED** [review]

**Comment by jasonblais** [comment]
@justinegeffen what are the next steps for the PR, or would this one be ready to merge?

**Comment by justinegeffen** [comment]
It's ready to be merged! Thank you. :)

**Pull request merged**